Sources/ArgumentParser/Parsable Properties/ArgumentVisibility.swift

```diff
@@ -39,8 +52,7 @@ extension ArgumentVisibility {
     }
   }
 
-  /// - Returns: true if `self` is at least as visible as the supplied argument.
-  func isAtLeastAsVisible(as other: Self) -> Bool {
```

Comment:
I actually prefer this API over the comparable conformance, especially since we eventually want to include per-group visibility, which won't be comparable in nature. A function call like this is more explicit about the relationship between the two visibilities, too. (I can kind of picture the ordering going in either direction.)

Reply:
Should be fixed.